tests/ttnn/unit_tests/operations/test_batch_norm.py

```diff
@@ -10,6 +10,127 @@
+    batch_mean_torch = torch.full(torch.Size([channel_size]), 0.00030171126, dtype=torch.float32)
+    batch_var_torch = torch.full(torch.Size([channel_size]), 0.1262342343, dtype=torch.float32)
+    weight_torch = torch.full(torch.Size([channel_size]), 0.246943565369, dtype=torch.float32) if weight else None
+    bias_torch = torch.full(torch.Size([channel_size]), 0.59, dtype=torch.float32) if bias else None
+
+    result_torch = torch.nn.functional.batch_norm(
+        input=input_tensor_torch,
+        running_mean=batch_mean_torch,
+        running_var=batch_var_torch,
+        weight=weight_torch,
+        bias=bias_torch,
+        eps=eps,
+    )
+
+    batch_mean_torch = batch_mean_torch.view(1, channel_size, 1, 1)
+    batch_var_torch = batch_var_torch.view(1, channel_size, 1, 1)
+    weight_torch = weight_torch.view(1, channel_size, 1, 1) if weight else None
+    bias_torch = bias_torch.view(1, channel_size, 1, 1) if bias else None
+
+    input_tensor_tt = ttnn.from_torch(input_tensor_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    batch_mean_tt = ttnn.from_torch(batch_mean_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    batch_var_tt = ttnn.from_torch(batch_var_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    weight_tt = (
+        ttnn.from_torch(weight_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device) if weight else None
+    )
+    bias_tt = ttnn.from_torch(bias_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device) if bias else None
+
+    result_tt = ttnn.batch_norm(
+        input_tensor_tt, running_mean=batch_mean_tt, running_var=batch_var_tt, eps=eps, weight=weight_tt, bias=bias_tt
+    )
+    tt_out = ttnn.to_torch(result_tt)
+
+    status_1 = torch.allclose(result_torch, tt_out, atol=1e-10, rtol=1e-5)
+    status_2 = compare_results_batch_norm([result_torch], [tt_out])
+    assert status_2 & status_1
+
+
+@skip_for_grayskull("Unsupported dtype for Grayskull")
+@pytest.mark.parametrize(
+    "input_shapes",
+    [
+        *(torch.Size([n, c, 32, 32]) for n, c in product([1, 2, 3, 4], [1, 2, 3, 4])),
+        *(torch.Size([n, c, 23, 23]) for n, c in product([1, 2, 3, 4], [1, 2, 3, 4])),
+        *(torch.Size([n, c, 64, 120]) for n, c in product([1, 2], [1, 2, 3])),
+        torch.Size([3, 1, 64, 120]),
+        torch.Size([3, 2, 64, 120]),
+    ],
+)
+@pytest.mark.parametrize(
+    "training, check_mean, check_var, testing_dtype",
+    [
+        (False, False, False, "float32"),  # xfail case
+        (False, True, False, "float32"),  # xfail case
+        (False, False, True, "float32"),  # xfail case
+        (False, True, True, "float32"),
+    ],
+)
+@pytest.mark.parametrize("weight", [True, False])
+@pytest.mark.parametrize("bias", [True, False])
+@pytest.mark.parametrize("eps", [1.0, 0.0, 2.34, 1e-05])
+def test_batch_norm_fp32(input_shapes, training, check_mean, check_var, weight, bias, eps, testing_dtype, device):
```

Comment:
Can we just remove this `training` parameter from the PR if we're not going to test it? then just set hardcoded `training=False` in the `ttnn.batch_norm` and `torch.nn.functional.batch_norm` calls below.

Reply:
Also remove `testing_dtype` and just hardcode that to `"float32"` in the calls to `data_gen_with_range_batch_norm` below. The test is called `test_batch_norm_fp32` after all.